functions/backup.bash

```diff
@@ -1,5 +1,32 @@
 #!/usr/bin/env bash
 
+# WORK IN PROGRESS
+filebrowser() {
+  bkppath=/var/lib/openhab2/backups
+  # shellcheck disable=SC2164,SC2012
+  bkpfile=$(cd ${bkppath}; ls -lthp openhab2-backup-* | head -20 | awk -F ' ' '{ print $9 " " $5 }')
+}
+
+backup_openhab_config() {
+  infotext="You have successfully created a backup of your openHAB configuration."
+  out=$(openhab-cli backup | tail -2 | head -1 | awk -F ' ' '{print $NF}')
+  msg="${infotext}\\n\\nFile is ${out}"
+  whiptail --title "Created openHAB config backup" --msgbox "$msg" 11 78
+}
+
+restore_openhab_config() {
+  filebrowser
+  # shellcheck disable=SC2086
+  fileselect=$(whiptail --title "Restore openHAB config" --cancel-button Cancel --ok-button Select  --menu "\\nSelect your backup from most current 20 files below:" 30 60 20 $bkpfile 3>&1 1>&2 2>&3)
```

Comment:
cancel button is ignored, pls add checking the result, do not restart OH in this case

Reply:
done